### ISSUE TITLE
Keyboard Tooltips Inconsistent for layer name

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#3819]
Tooltips doesn't behave the same way with mouse and keyboard

If I mouse over a layer name, I can see a tooltip with the layer name. Keyboard doesn't get focus on layer name and no tooltip is shown. 

Keyboard focus should have same tool tip as mouse

**Comment by AleksueiR** [comment]
Yeap, it looks like we have the tooltips set on non-focusable `div` containers wrapping layer items, not on the focusable buttons themselves. I hope the fix could be as simple as moving the tool tip directive to the actual `button` node, although we might need to tweak `aria-label`s already present on `button` nodes.

**Comment by RyanCoulsonCA** [comment]
@AleksueiR  Moving the tool tip into the button node works, however some layers aren't added to the legend as a button, which means they won't be focusable when tabbing through (and therefore won't show the tool tip). This is an example of one of those layers:
![image](https://i.imgur.com/2vdTieG.gif)

The same thing occurs for the `Electric Transmission Line` layer in the group below it.

One solution is to add these layers to the legend as a button and have them do nothing on click.
![](https://i.imgur.com/WvWzt7Q.gif)

Is this the solution I should go with, or is there another one that may be easier?

**Comment by AleksueiR** [comment]
hm, for the layers which are not added as buttons, are they all disabled layers? Is this conditional on a single template, or is there a separate template for such layers? If so, can we mark their containers as tabbable with `tabindex` attribute?

**Comment by RyanCoulsonCA** [comment]
The `Electric Transmission Line` layer has `"controls": ["remove"]` in its config which seems to be the reason why it doesn't get added as a button, and the layers that don't load (like the `spatial density of major ...` layer in the gifs above) have their own template.

I tried adding `tabindex` to the container but it doesn't seem to work (I may be doing this wrong, I'm not too familiar with it)

**Comment by AleksueiR** [comment]
> I tried adding tabindex to the container but it doesn't seem to work (I may be doing this wrong, I'm not too familiar with it)

Could be that Focus Manager interferes somehow. @milespetrov if you have a minute, could you take a look if we can make the layer item container tabbable?

**Comment by RyanCoulsonCA** [comment]
I'll PR the fix for the buttons in the meantime.

**Comment by jolevesq** [comment]
@dan-bowerman Is this will be part of 3.3.4?
